src/option.c

```diff
@@ -212,3 +212,24 @@ get_alias_value(const char *address)
     return address;
 }
 
+const char *
+get_alias_value_ext(const char *address, int port)
+{
+    int i;
+
+    if (!address)
+        return NULL;
+
+    char *addr = sng_malloc(1024);
```

Comment:
I cant locate where is memory allocated here freed.
This could cause a memory leak

Reply:
thanks @Kaian 
I remade code a little according to your comment
Please take a look.